iib/workers/tasks/build_merge_index_image.py

```diff
@@ -344,3 +345,13 @@ def is_bundle_version_valid(bundle_path, valid_ocp_version):
         )
 
     return False
+
+def correct_float_precision(version):
```

Comment:
Maybe rename it to simply `float_precision` and have the precision as a parameter which defaults to 2?

Reply:
sure